test/integration/aens.ts

```diff
@@ -176,15 +177,15 @@ describe('Aens', function () {
       const claim = await preclaim.claim()
```

Comment:
**line 157:** throws an `Returning an awaited promise is required in this context.` error. Should be `return await...`